actors/builtin/reward/reward_state.go

```diff
@@ -3,305 +3,59 @@ package reward
 import (
 	abi "github.com/filecoin-project/specs-actors/actors/abi"
 	big "github.com/filecoin-project/specs-actors/actors/abi/big"
-	builtin "github.com/filecoin-project/specs-actors/actors/builtin"
-	adt "github.com/filecoin-project/specs-actors/actors/util/adt"
 )
 
-// Fractional representation of NetworkTime with an implicit denominator of (2^MintingInputFixedPoint).
-type NetworkTime = big.Int
-
 // A quantity of space * time (in byte-epochs) representing power committed to the network for some duration.
 type Spacetime = big.Int
 
 type State struct {
-	BaselinePower        abi.StoragePower
-	RealizedPower        abi.StoragePower
-	CumsumBaseline       Spacetime
-	CumsumRealized       Spacetime
-	EffectiveNetworkTime NetworkTime
+	// CumsumBaseline is a target CumsumRealized needs to reach for EffectiveNetworkTime to increase
+	CumsumBaseline Spacetime
+	CumsumRealized Spacetime
 
-	SimpleSupply   abi.TokenAmount // current supply
-	BaselineSupply abi.TokenAmount // current supply
+	// EffectiveNetworkTime is ceiling of EffectiveNetworkTime based on CumsumRealized
+	EffectiveNetworkTime abi.ChainEpoch
 
-	// The reward to be paid in total to block producers, if exactly the expected number of them produce a block.
+	// The reward to be paid in per WinCount to block producers.
 	// The actual reward total paid out depends on the number of winners in any round.
-	// This is computed at the end of the previous epoch, and should really be called ThisEpochReward.
 	ThisEpochReward abi.TokenAmount
-
-	// The count of epochs for which a reward has been paid.
-	// This should equal the number of non-empty tipsets after the genesis, aka "chain height".
-	RewardEpochsPaid abi.ChainEpoch
+	Epoch           abi.ChainEpoch
 }
 
-type AddrKey = adt.AddrKey
-
-// These numbers are placeholders, but should be in units of attoFIL, 10^-18 FIL
-var SimpleTotal = big.Mul(big.NewInt(100e6), big.NewInt(1e18))   // 100M for testnet, PARAM_FINISH
-var BaselineTotal = big.Mul(big.NewInt(900e6), big.NewInt(1e18)) // 900M for testnet, PARAM_FINISH
-
-func ConstructState(currRealizedPower *abi.StoragePower) *State {
+func ConstructState(currRealizedPower abi.StoragePower) *State {
 	st := &State{
-		BaselinePower:        big.Zero(),
-		RealizedPower:        big.Zero(),
-		CumsumBaseline:       big.Zero(),
-		CumsumRealized:       big.Zero(),
-		EffectiveNetworkTime: big.Zero(),
+		CumsumBaseline: big.Zero(),
+		CumsumRealized: big.Zero(),
```

Comment:
Init `EffectiveNetworkTime`, even if it will be overwritten.

Nit: Init `Epoch` here rather than as a mutation later?